types/p-lazy/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for p-lazy 1.0
+// Project: https://github.com/sindresorhus/p-lazy#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = PLazy;
+
+declare class PLazy<T> extends Promise<T> {
+    static from<T>(fn: () => T | PromiseLike<T>): PLazy<T>;
```

Comment:
Isn't it inherited from the Promise constructor? When I try it out in the tests, it is being correctly detected when the constructor parameter is missing. Furthermore,  the resolve() and reject() callbacks passed as parameters to the executor fn are also correctly typed in the test.